frontend/packages/console-app/src/components/nodes/NodeStatus.tsx

```diff
@@ -1,16 +1,67 @@
 import * as React from 'react';
+import * as _ from 'lodash';
+import ConsumerPopover from '@console/shared/src/components/dashboard/utilization-card/TopConsumerPopover';
 import { Status, SecondaryStatus, getNodeSecondaryStatus } from '@console/shared';
 import { NodeKind } from '@console/internal/module/k8s';
+import { humanizeBinaryBytes, humanizeNumber } from '@console/internal/components/utils';
 import { nodeStatus } from '../../status/node';
+import { PressureQueries, Condition } from '../../queries';
+
+const conditionDescriptionMap = Object.freeze({
+  [Condition.DISK_PRESSURE]: 'available memory is low',
+  [Condition.MEM_PRESSURE]: 'available disk capacity is low',
+  [Condition.PID_PRESSURE]: 'CPU is running a large number of processes',
```

Comment:
I'll be honest, PID pressure is new to me.

Reply:
The only pressure I know is `blood pressure`. However, K8s supports one more condition `Network Unavailable` but I cannot find it in Openshift. I asked around in #form-node couldn't get any answers... :/

Reply:
😄 I think these 3 would be fine for now

Reply:
Plz add `Condition.BLOOD_PRESSURE`.